redfish/provider/resource_redfish_bios.go

```diff
@@ -333,13 +361,20 @@ func (r *BiosResource) readRedfishDellBiosAttributes(service *gofish.Service, d
 	}
 
 	attributesTF := make(map[string]attr.Value)
-	for key, value := range attributes {
-		if _, ok := old[key]; ok {
+	if len(old) != 0 {
```

Comment:
This will not distinguish between an empty map and Null value.
Maybe we are looking for a nil here?
```suggestion
	if old != nil {
```